etc/org.opencastproject.serviceregistry.impl.ServiceRegistryJpaImpl.cfg

```diff
@@ -1,9 +1,13 @@
 # Configuration for the service registry
 
-# Number of failed jobs on a service before to set it in error state.
+# Number of failed jobs on a service before to set it in error state. Set to -1 to disable error states completely.
 # Default: 10
 #max.attempts=10
 
+# Comma-separated list of services that should never go into error state.
```

Comment:
can you put an example what to specify as the service

Reply:
Added example and clarified config key. :)